Gemfile

```diff
@@ -32,7 +32,6 @@ end
 
 group :development, :test do
   gem "capybara"
-  gem "capybara-accessible"
```

Comment:
I removed this because I copied form helper methods from GCF which were raising accessibility issues. I think we should re-add this and fix this, but it felt out of scope for this. Created a card here:  https://trello.com/c/4zn71DZM